specifications/factoring/factoring.yml

```diff
@@ -1524,6 +1532,12 @@ components:
               type: number
               description: The number of days the claim full payment is overdue. If the number is negative, the due date of the payments is still in the future. If the number is positive, the claim is now late.
               example: 5
+            currency:
+              type: string
+              description: Currency for claim in ISO 4217 format
+
+
```

Comment:
You have a lot of empty lines here :grin:

Reply:
:scream: